apps/barista/src/app.module.ts

```diff
@@ -82,6 +83,7 @@ import { DtToastModule } from '@dynatrace/barista-components/toast';
     DtThemingModule,
     DtExamplesModule,
     DtToastModule,
+    RouterModule.forRoot([], { errorHandler: () => {} }),
```

Comment:
Could you please add a comment on why we did this?

Reply:
That would be interesting because the app should not depend on the router module